test/functional/dataElements/xdmObject.spec.js

```diff
@@ -123,6 +109,54 @@ test("initializes form fields with individual object attribute values", async ()
   await stringEdit.expectValue("Adobe");
 });
 
+test("disables user from selecting a sandbox", async () => {
+  // temporarily remove sandboxes mock
+  await t.removeRequestHooks(platformMocks.sandboxes);
+  // replace with unaurhotized mock
+  await t.addRequestHooks(platformMocks.sandboxesEmpty);
+  await initializeExtensionView();
+  await spectrum.select("sandboxField").expectDisabled();
+  await selectSchemaFromSchemasMeta();
+  await xdmTree.toggleExpansion("_alloyengineering");
+  // restore original sandboxes mock
+  await t.addRequestHooks(platformMocks.sandboxes);
```

Comment:
Ideally, tests should be atomic, meaning if one test blows up it doesn't blow up other tests. If this test blows up (e.g., an error is thrown), this last line may not be called, causing subsequent tests to also fail. It might be better to do `await t.addRequestHooks(platformMocks.sandboxes);` inside a `beforeEach` rather than calling `.requestHooks()` on the fixture and having this line here.

Reply:
Yes, I see the problem now. I will make the change